Add release notes for June 2020 frontend release (#430)
SenseNet/sensenet.github.io: _updates/2020-06-04-june.md

---
title: June 2020
---

## Summertime, and the livin' is easy 😎☀️

The weather is getting warmer but hard work never stops. We are still working on making our product better and better. In the last 4 weeks we were working on performance optimization, layout standardization and revamp our example apps to be compatible with the new authentication.
So here are the changes what we made:

### Admin-ui@1.8.0

> - We tried to reduce the numbers of request, the unnecessary requests have been removed, now we have almost half of it then before ⭐[#672](https://github.com/SenseNet/sn-client/pull/672)

Author: ```suggestion
- We tried to reduce the numbers of requests, unnecessary ones have been removed and at the end we were able to halve the numbers ⭐[#672](https://github.com/SenseNet/sn-client/pull/672)
```

> - The command palette has been changed a lot. Its new goal is to be a 'global searchfield' where you can search in all of the contents in the current repository easily. With the new solution it is faster 🚀 and smoother 💨 than ever before. Of course the [main features](https://docs.sensenet.com/guides/search/03-command-palette) of the old version are still working, like: content query, custom actions and help function. [#685](https://github.com/SenseNet/sn-client/pull/685)

Author: ```suggestion
- The command palette has changed a lot. Its new goal is to be a 'global search field' where you can search in all of the contents in the current repository easily. With the new solution it is faster 🚀 and smoother 💨 than ever before. Of course the [main features](https://docs.sensenet.com/guides/search/03-command-palette) of the old version are still working, like: using content queries, custom actions and help functions. [#685](https://github.com/SenseNet/sn-client/pull/685)
```

  ![Search](/img/posts/search.gif "Search")

> - Layout standardization [#669](https://github.com/SenseNet/sn-client/pull/669): Cause the visibility of the tree and the breadcrumb was a little bit confusing we wanted to make them standardized.

Author: ```suggestion
- Layout standardization [#669](https://github.com/SenseNet/sn-client/pull/669): Cause the visibility of the tree and the breadcrumb was a little bit confusing we wanted to unify them.
```

> Users and groups has a tree from now, if the "root" has a folder content, otherwise the tree does not appear.

Author: ```suggestion
In the _Users and groups_ menu, if the "root" has not only users and children, but also container contents like folders, you can use the same tree for navigating as in the _Content_ menu, a otherwise the tree does not appear.
```

  ![Tree](/img/posts/tree.gif "Tree")

- On the edit/browse form the icon in the title is now a link to the related CTD

  ![IconLink](/img/posts/iconLink.gif "IconLink")

### @sensenet/redux@5.5.0 🆕

- New method is available on stores created by createSensenetStore function: **reloadRepository**
With this function you can change the current repository in store runtime. It would be useful after authentication 🛂, if you need a new repository instance. [#686](https://github.com/SenseNet/sn-client/pull/686)

### @sensenet/redux-promise-middleware@2.2.0 🆕

- New method has been created in the middleware as well: **reset**. We needed it cause of the change in @sensenet/redux package ☝️ 

### Dear contributors!
We have a new ESLint rule for ordering imports [#684](https://github.com/SenseNet/sn-client/pull/684)
The order:
- @sensenet/... packages
- @material-ui/... packages
- any other packages

> It should throw error, so it is mandatory to keep it to every contributor of the project.

Author: ```suggestion
```
